README.md

```diff
@@ -132,6 +132,15 @@ Options:
 
 ```
 
+#### Slack Notifications
+
+To enable Slack notifications during the deploy process create a [Slack Webhook](https://api.slack.com/incoming-webhooks) and add two entries `SLACK_WEBHOOK` and `SLACK_CHANNEL` to your `env.yml`.
+
+```
+SLACK_CHANNEL: '#devops'
```

Comment:
@evansiroky it just gets passed to https://github.com/clonn/slack-node-sdk so I guess they figure it out there.